### PR TITLE
Add progress indicator to featurization

### PR DESCRIPTION
**Pull request by stefanv**
Addresses #244 as far as featurization is concerned

@profjsb Did you also want progress bars for model building and prediction?

### CODE REVIEW

File: static/js/reducers.js [hunk 1]
```diff
@@ -40,6 +40,20 @@ function featuresets(state=[], action) {
...
+      const newState = [ ...state ];
+
+      const { percent, elapsed } = { ...action.payload };
+      const featureIdx = newState.findIndex((element) => (
+        element.id == action.payload.fsetID
+      ));
+
+      if (featureIdx != -1) {
+        const feature = newState[featureIdx];
+        feature.progress = {percent, elapsed};
```
Comment (acrellin) [inline static/js/reducers.js]: Just spacing here: `{ percent, elapsed }`

**Review by acrellin — APPROVED** [review]
Looks great! Just a minor syntax comment, otherwise looks good to go :+1:

**Comment by acrellin** [comment]
Also, `dask` needs to be bumped to `0.17.0` in `cesium` for this to work.

**Comment by stefanv** [comment]
Thanks, fixed!

**Comment by stefanv** [comment]
See also https://github.com/cesium-ml/cesium/pull/267

**Pull request merged**